apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -86,6 +68,84 @@ export default function HomePage() {
         }}
       >
         <ELGettingStarted />
+
+        <div className={'home-bg-dark'}>
+          <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
+            <FeaturedDataTabs
+              sql={dataSql}
+              rgbIndex={3}
+              configs={[
+                {
+                  title: 'Human Studies',
+                  icon: 'PERSON',
+                  explorePagePath: '/Explore/Studies',
+                  exploreObjectType: 'Studies',
+                  plotsConfig: {
+                    configs: [
+                      {
+                        title: 'The Long Life Family Study',
+                        description:
+                          'The Long Life Family Study (LLFS) investigates genetic and familial factors in exceptional longevity. Families were recruited based on a Family Longevity Selection Score (FLOSS) of ≥7. Over 4,953 individuals from 539 families were phenotyped through in-home visits in the U.S. and Denmark with centralized assays and standardized protocols.',
+                        facetsToPlot: ['dataTypes'],
+                        selectFacetColumnName: 'Study',
+                        selectFacetColumnValue: 'LLFS',
+                        detailsPagePath:
+                          '/Explore/Studies/DetailsPage?studyKey=LLFS',
+                        unitDescription: 'Files',
+                        plotType: 'STACKED_HORIZONTAL_BAR',
+                      },
+                      {
+                        title: 'ADAMTS7 Study',
+                        description: '',
```

Comment:
Add config to ELITE home page v2.  Descriptions (and potentially titles) will need to be updated.